VAMobile/src/components/DefaultList.tsx

```diff
@@ -22,6 +23,8 @@ export type DefaultListProps = {
   items: Array<DefaultListItemObj>
   /** if true the text will be selectable */
   selectable?: boolean
+  /** optional accessibility role */
+  a11yRole?: AccessibilityRole
```

Comment:
Is this needed, this prop isn't being used anywhere in the component?

Reply:
Likely not. I was using it to test whether by adding role to the list it will remove the interactive flag for DOB